include/nfc/ndef/file.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef NFC_NDEF_FILE_H_
+#define NFC_NDEF_FILE_H_
+
+/**@file
+ *
+ * @defgroup nfc_ndef_file NFC NDEF File
+ * @{
+ * @brief Generation of NFC NDEF File for the NFC Type 4 Tag.
+ *
+ */
+
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Size of NLEN field, used to encode NDEF message for Type 4 Tag. */
+#define NFC_NDEF_FILE_NLEN_FIELD_SIZE 2U
```

Comment:
Could we move it to the source file? A user of this library might be tempted to retrieve NDEF message from NDEF file using this offset instead of using ``nfc_ndef_file_msg_get`` API

Reply:
Done